index.src.html

```diff
@@ -655,6 +655,32 @@ <h2 id="integration-with-html">Integration with HTML</h2>
   with W3C HTML5, those would need to be copied from [[HTML]].
 </section>
 
+<section class="informative">
+  <h2 id="integration-with-css">Integration with CSS</h2>
+
+  The CSS Standard does not specify how it fetches resources referenced from
+  stylesheets. However, implementations use the URL a given stylesheet was
+  loaded from as referrer for the requests for resources from that stylesheet.
+
+  Implementations should keep track of a referrer policy for each stylesheet
+  that should be used to create requests for resources from the respective
+  stylesheet. 
+
+  For external stylesheets, the referrer policy should be
+  <a>"<code>no-referrer-when-downgrade</code>"</a> unless overwritten by an
+  `<code>Referrer-Policy</code>` header.
+
+  Note: If the stylesheet was loaded via a HTML <{link}> element with a
+  declared <{link/referrerpolicy}>, this referrer policy will not affect the
+  requests for resources referenced from the stylesheet.
+
+  For inline stylesheets, the referrer policy is the containing {{Document}}'s
+  referrer policy.
```

Comment:
Maybe it's best to say here when it's captured? Or merge the next paragraph into this one.